api/src/main/scala/dcos/metronome/api/ApiModule.scala

```diff
@@ -16,46 +17,37 @@ import mesosphere.marathon.core.plugin.PluginManager
 import mesosphere.marathon.metrics.Metrics
 import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer }
 import play.api.http.HttpErrorHandler
-import play.api.mvc.{ AnyContent, BodyParser }
+import play.api.mvc.ControllerComponents
 import play.api.routing.Router
 import router.Routes
 
 import scala.concurrent.ExecutionContext
 
-class ApiModule(
+class ApiModule(controllerComponents: ControllerComponents, assets: Assets, httpErrorHandler: HttpErrorHandler)(
+  implicit
+  ec:                 ExecutionContext,
```

Comment:
Why the change for `ec` from implicit?

Reply:
`ec` was already previously `implicit` (Updated due to deprecated play ec import)

Reply:
I'll mark this as resolved as well